Projects/Checkers/Game.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Checkers;
+
+public class Game
+{
+	private const int PiecesPerColor = 12;
+
+	public PieceColor Turn { get; private set; }
+	public Board Board { get; private set; }
```

Comment:
Don't require a setter if it is set up in the constructor

Reply:
fixed in 880198f8088f584ce70c31b053d68bdbe5bd576b